build.gradle

```diff
@@ -183,6 +185,9 @@ dependencies {
     compile("org.apache.ivy:ivy:$ivyVersion") {
         transitive = false
     }
+    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$jacksondataformatyamlVersion"
+    compile "com.fasterxml.jackson.core:jackson-databind:$jacksondatabindVersion"
+
```

Comment:
Just curious why these 2 deps are listed here and not in `groovy-yaml/build.gradle`?